signup-page.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<style>
+    .main-container{
+        background-color: #EDF0F5;
+        padding: 1% 3%;
+        margin: 3% 35%;
+        font-size: 18px;
+        font-family:Georgia, 'Times New Roman', Times, serif;
+            }
+    .btn-singup{
+        background-color: #5E9B4D;
+        margin: 20px 0px;
+        padding: 10px 50px;
+        border: 2px solid #5E9B4D;
+        border-radius: 5px;
+        font-weight: bold;
+        font-size: 20px;
+        color: white;
+    }
+    .textStyle{
+        font-weight: bold;
+        color: #BEC3C9;
+      
+    }
+    .textInput{
+        width: 40%;
+        height: 35px;
+        border-radius: 5px;
+        margin: 5px 0px;
+        font-size: 20px;
+        border:none;
+        padding: 1% 3%;
+        }
+    .inputMobileAndPassword{
+        width: 90%;
+    }
+    select{
```

Comment:
Avoid using any html tag as selector, as you can grow your code later and add a new tag of that sort without realizing where the styles come from. Bad practice.